src/Microsoft.IdentityModel.Validators/AadIssuerValidator/AadIssuerValidator.cs

```diff
@@ -264,7 +264,7 @@ private static string GetTenantIdFromToken(SecurityToken securityToken)
                 if (jsonWebToken.TryGetPayloadValue(AadIssuerValidatorConstants.Tid, out string tid))
                     return tid;
 
-                if (jsonWebToken.TryGetPayloadValue(AadIssuerValidatorConstants.Tid, out string tenantId))
+                if (jsonWebToken.TryGetPayloadValue(AadIssuerValidatorConstants.TenantId, out string tenantId))
```

Comment:
>TenantId [](http://example.com/codeflow?start=80&length=8)

Was the old code a mistake? It was getting Tid twice, as tid and tenantId.

Reply:
@dannybtsai there are two possibilities for the 'tid' claim, therefore there are two constants.